week-9/Homework/mandatory/2-exercises/4-music-player.js

```diff
@@ -21,21 +21,60 @@ This means the order the songs are played in will be random, but each song will
 
 class MusicPlayer {
     // Add your code here
+    constructor(){
+        this.playlist = [];
```

Comment:
Is it possible to have a single array like `playList` that stores in the information about the title and the artist ? Think of another data type you could use other than a string